src/domains/issue/service/issue.service.impl.ts

```diff
@@ -1,55 +1,46 @@
 import { type IssueService } from '@domains/issue/service/issue.service';
 import { type IssueRepository } from '@domains/issue/repository';
-import { type ProjectRepository } from '@domains/project/repository';
-import { type IssueDataDTO, type IssueDTO } from '@domains/issue/dto';
-import { db, NotFoundException } from '@utils';
-import { type ProjectDTO } from '@domains/project/dto';
-import type { ProjectManagementToolAdapter } from '@domains/adapter/projectManagementToolAdapter';
-import { type UserDTO, type UserRepository } from '@domains/user';
+import { type EventRepository } from '@domains/event/repository';
+import { type ManualTimeModificationDTO, type TimeTrackingDTO } from '@domains/event/dto';
+import { ConflictException, NotFoundException } from '@utils';
+import { type IssueDTO } from '@domains/issue/dto';
+import { getTimeTrackedInSeconds } from '@utils/date-service';
 
 export class IssueServiceImpl implements IssueService {
   constructor(
-    private readonly adapter: ProjectManagementToolAdapter,
     private readonly issueRepository: IssueRepository,
-    private readonly projectRepository: ProjectRepository,
-    private readonly userRepository: UserRepository
+    private readonly eventRepository: EventRepository
   ) {}
 
-  async integrateProjectIssues(projectId: string): Promise<IssueDTO[]> {
-    const project: ProjectDTO | null = await this.projectRepository.getById(projectId);
-    if (project === null) {
-      throw new NotFoundException('Project');
+  /**
+   * Retrieves the total worked time in seconds for a given issue.
+   * This method calculates the total time tracked automatically and manually
+   * for the specified issue.
+   *
+   * @param {string} issueId - The ID of the issue to retrieve worked time for.
+   * @returns {Promise<number>} A promise that resolves to the total worked time in seconds.
+   * @throws {NotFoundException} If the specified issue cannot be found.
+   * @throws {ConflictException} If the calculated worked time is negative.
+   */
+  async getIssueWorkedSeconds(issueId: string): Promise<number> {
+    const issue: IssueDTO | null = await this.issueRepository.getById(issueId);
+    if (issue === null) {
+      throw new NotFoundException('Issue');
     }
+    let workedTime: number = 0;
```

Comment:
No need to add `number` type. 0 is a number, so type is already inferred.

Reply:
done